### DISCUSSION TITLE
White Balance differences

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#634] [General]
This one has been stumping me for a while.

When we define a set White Balance value... i.e. 6000K, I'm observing there are significant color variations between the cameras using the same video profiles. The color on the HERO12 is noticeably different to the HERO11.

After looking at the GPMF streams, I can see the WRGB values are completely different... even between the HERO11 and HERO11 Mini.
I understand there may be variations between the sensors, but all three use the same sensor in this case. So why the difference, even between HERO11 Mini and HERO11?

**Comment by dnewman-gpsw** [comment]
No idea. Camera tuning is ongoing. I tested H12 and H11mini, and 6000K differed  H12 (2.1367, 1.0000, 1.5195) and H11Mini (2.1367, 1.0000, 1.5352), so Mini has slightly more blue gain.  I also noticed that Mini used a slightly higher ISO for the same scene.  There are sensors differences, no two sensors are identical, maybe there is some per camera calibration (totally guessing.)

**Reply by RikshaDriver** [reply]
Completely understand regarding silicon variations, but for video one would assume a common calibration constant is normally defined?

on my HERO12, 6000K is (2.137, 1.000, 1.578) which is different from your values

Looking at some other 6500k files I have on hand:

HERO11:  2.223, 1.000, 1.395
HERO11 Mini:  2.270, 1.000, 1.359
HERO12:  2.184, 1.000, 1.418

Would be interesting to get some understanding on why there are differences between the cameras, especially for defined presets.

**Reply by dnewman-gpsw** [reply]
I know we have had per camera calibrations on cameras like MAX.  All I can think of.
